examples/visualize_optimizers.py

```diff
@@ -16,16 +16,18 @@
 
 filterwarnings('ignore', category=UserWarning)
 
+IMG_FORMAT = 'jpg'
 OPTIMIZERS_IGNORE = ('lomo', 'adalomo', 'demo', 'a2grad', 'muon', 'alice', 'adamc', 'adamwsn', 'adamuon', 'splus')
 OPTIMIZERS_MODEL_INPUT_NEEDED = ('lomo', 'adalomo', 'adammini')
 OPTIMIZERS_GRAPH_NEEDED = ('adahessian', 'sophiah')
 OPTIMIZERS_CLOSURE_NEEDED = ('alig', 'bsam')
-EVAL_PER_HYPERPARAM: int = 540
-OPTIMIZATION_STEPS: int = 300
-TESTING_OPTIMIZATION_STEPS: int = 650
+EVAL_PER_HYPERPARAM: int = 600
+OPTIMIZATION_STEPS: int = 200
+TESTING_OPTIMIZATION_STEPS: dict[str, int] = {'rastrigin': 150, 'rosenbrock': 400}
```

Comment:
Could you please make a type hint using `typing` module (e.g. from typing import Dict) instead of using built-in type? cause this script could be run on old version of Python too!